### ISSUE TITLE
Utiliser ACR122u-a9

### ISSUE THREAD

**Issue by Kyosuke33** [xavave/Mifare-Windows-Tool#62]
Bonjour, j'ai voulu me lancer avec un ACR122u-a9 pour copier qq tag de mon appart et du boulot.
Avant d'utiliser votre programme, l'ACR122u-9 s'allumé en rouge et quand j y mettais une carte il passait vert.
Maintenant il n'y a plus aucune lumière. Si je pose le tag et que je met lire, ca le trouve, mais il n'y a plus de lumière ou de son sur l'acr122u-a9.
Quand je met lire un tag, il y a une fenetre noir qui s ouvre, ou je peut cocher ou non des .keys et autres, que faut il utiliser.
Y a il un tuto complet pour l'utilisation qq part ? (je l'ai peut etre raté)

Merci.
![ca8d028fe8d249fb1ddb43e5b5f1e77d](https://user-images.githubusercontent.com/44751932/212745878-0eae70a4-e9bf-4948-82de-8cda4a2d05db.png)

**Comment by xavave** [comment]
Bonjour, 
Je ne suis pas sur que l'ACR122U-A9 puisse fonctionner.
En tout cas c'est normal en effet, que l'ACR n'ait pas de lumière ni de BIP, c'est parce que le pilote LibUsbK qui ne gère ni la LED ni le bip.
Il y a un tuto dans le wiki : https://github.com/xavave/Mifare-Windows-Tool/wiki/Accueil

**Comment by Kyosuke33** [comment]
Ok merci. Pour le moment quand je lance lire le tag, en ayant coché les 3 extension keys possible (je savais pas si il fallais le faire ou pas) , ca travail pendant 5 minutes et a la fin j'ai une fenêtre avec un font noir ou je peut écrire le tag, avec des options a cocher comme écrire bloc 0... Jv essayer, on vera bien. C'est la version releas me 1.5 que j'ai installé, pas les dernières 1.6

**Comment by xavave** [comment]
Oui c'est le comportement normal. Lisez bien le wiki si besoin

**Comment by Kyosuke33** [comment]
Quand j'essaye de changer l'UID du bloc 0 de ma carte qui a une bloc 0 reinscriptible, j'ai ce message "Warning: Unlock command [1/2]: failed / not acknowledged."
Vous auriez une idée de la raison ? 
Merci

**Comment by xavave** [comment]
Vous avez lu la doc ?

https://github.com/xavave/Mifare-Windows-Tool/wiki/Write-Tag-fr-FR#user-content-clone-tag

**Comment by Kyosuke33** [comment]
Oui, j'ai lu et fait ce qui est demandé, mais même message. J'ai testé les 2 options de reinscription bloc 0 chinoi gen 2 et l'autre mais ca fonctionne pas.
Je viens d essayer l'ancienne version du logiciel, même message.
J'ai aussi essayé la toute dernière, même message, avec en plus un autre "probleme" c'est le lecteur ACR qui se deconnect tout seul, alors que ca le fait pas sur les autres versions.
Merci pour l'aide je vais aller chercher des infos un peu partout pour y arrive.

**Comment by Kyosuke33** [comment]
I just see there is the same issue open. I have a 7 bytes card. i'm going to read the discussion^^
